.env-dist

```diff
@@ -76,3 +76,6 @@ PRODUCT_PROMOS_ENABLED=1
 
 # Experiment Flag
 EXPERIMENT_ACTIVE=0
+
+REDIS_URL=redis://127.0.0.1:6379
+MAX_NUM_ADDRESSES=5
```

Comment:
This was hard coded to 15 below. Will it still be 15 in prod, or we dropping the limit?

Reply:
Dropping the limit to 5. 99.9% of our subscribers are below 5 addresses already.

Reply:
Sorry, [99.7%](https://bugzilla.mozilla.org/show_bug.cgi?id=1648582#c7).

Reply:
Interesting. What will be the impact to the 0.3% of the users who are **over** the new 5 address limit? Will they be able to keep their additional addresses, or will it randomly delete addresses so they fit within the new 5 address limit?

Reply:
They'll be able to keep all their existing addresses, but won't be able to add any new ones.

Reply:
Well I'm one of the 0.3% and wonder why you upgrade the system to an "unlimited" system when lowering the value to 5.
I'm really dissatisfied :(